spec/order_processor_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'order_processor'
+
+describe OrderProcessor do 
+
+	subject { OrderProcessor.new notifier, calculable }
+	let( :notifier ) { double 'notifier', :call => nil }
+
+	describe "#process" do
+		
+		context "when is accepted" do
+			it "pass call to notifier" do
+				expect( notifier ).to receive :call
+				notifier.call
+			end
+		end
+
+		context "when is denied" do
+			it "raises an error" do
+				#code
```

Comment:
Incorrect indentation detected (column 4 instead of 5).<br>Missing space after #.<br>Tab detected.